docbook/reference/en/en-US/modules/StringConverter.xml

```diff
@@ -665,5 +665,152 @@ public static class TestResource {
    }
    </programlisting>
    </sect1>
+ 
+    <sect1>
+   <title>Default multiple valued <classname>ParamConverter</classname></title>
    
+   <para>
+   RESTEasy includes two built-in <classname>ParamConverter</classname>s in the resteasy-jaxrs module,
+   one for <classname>Collection</classname>s:
+   </para>
+   
+<programlisting>
+   <classname>org.jboss.resteasy.plugins.providers.MultiValuedCollectionParamConverter</classname>,
+</programlisting>
+
+   <para>
+   and one for arrays:
+   </para>
+   
+<programlisting>
+   <classname>org.jboss.resteasy.plugins.providers.MultiValuedArrayParamConverter</classname>,
+</programlisting>
+   
+   <para>
+   which implement the concepts in the previous section.
+   </para>
+   
+   <para>
+   In particular, <methodname>MultiValued*ParamConverter.fromString()</methodname> can transform a
+   string representation coming over the network into a <classname>Collection</classname> or array, and
+   <methodname>MultiValued*ParamConverter.toString()</methodname> can be used by a client side proxy
+   to transform <classname>Collection</classname>s or arrays into a string representation.
+   </para>
+   
+   <para>
+   String representations are determined by <classname>org.jboss.resteasy.annotations.Separator</classname>,
+   a parameter annotation in the resteasy-jaxrs module:
+   </para>
+   
+<programlisting>
+@Target({ElementType.METHOD, ElementType.PARAMETER})

```

Comment:
You forgot to remove the `ElementType.METHOD ` here

Reply:
Yep. Thanks.